### PR TITLE
Added missing menu and toolbar items to list

### PR DESCRIPTION
**Pull request by tylerkelly13**

### CODE REVIEW

File: advanced/editor-control-identifiers.md [hunk 1]
```diff
@@ -38,11 +38,21 @@ Use the [toolbar]({{ site.baseurl }}/configure/editor-appearance/#toolbar) or th
...
-| insert | core | Groups various insert actions into one button. |
+| h1 | core | Changes current line to the "Heading 1" style. |
+| h2 | core | Changes current line to the "Heading 2" style. |
+| h3 | core | Changes current line to the "Heading 3" style. |
+| h4 | core | Changes current line to the "Heading 4" style. |
+| h5 | core | Changes current line to the "Heading 5" style. |
+| h6 | core | Changes current line to the "Heading 6" style. |
+| remove | core | Removes (deletes) the selected content or the content before the cursor position. |
+| selectall | core | Selects all content in the editor. |
 | hr | [hr]({{ site.baseurl }}/plugins/hr/) | Inserts a horizontal rule into the editor. |
```
Comment (lnewson) [inline advanced/editor-control-identifiers.md]: Atm this list seems in a rather random order, just grouped by plugin. Maybe we should reorder it so the plugins are in alphabetical order?

File: advanced/editor-control-identifiers.md [hunk 2]
```diff
@@ -90,26 +103,35 @@ Use the [toolbar]({{ site.baseurl }}/configure/editor-appearance/#toolbar) or th
 | fullpage | [fullpage]({{ site.baseurl }}/plugins/fullpage/) | Documents properties for the full page. |
 | ltr | [directionality]({{ site.baseurl }}/plugins/directionality/) | Sets the directionality of contents to ltr. |
 | rtl | [directionality]({{ site.baseurl }}/plugins/directionality/) | Sets the directionality of contents to rtl. |
+| casechange | [casechange]({{ site.baseurl }}/plugins/casechange/) | Changes the case of text in a block selection to uppercase, lowercase, or title case. |
 | emoticons | [emoticons]({{ site.baseurl }}/plugins/emoticons/) | Opens the emoticons dialog. |
 | template | [template]({{ site.baseurl }}/plugins/template/) | Inserts templates into the editor. |
-| forecolor | [textcolor]({{ site.baseurl }}/plugins/textcolor/) | Applies foreground/text color to selection. |
-| backcolor | [textcolor]({{ site.baseurl }}/plugins/textcolor/) | Applies background color to selection. |
 | restoredraft | [restoredraft]({{ site.baseurl }}/plugins/autosave/) | Restores to the latest auto saved draft. |
 | insertfile | [moxiemanager]({{ site.baseurl }}/plugins/moxiemanager/) | Opens the MoxieManager dialog. |
```
Comment (lnewson) [inline advanced/editor-control-identifiers.md]: This should probably be `Tiny Drive` now instead since we're pushing that instead of MoxieManager.

Comment (tylerkelly13) [reply advanced/editor-control-identifiers.md]: yep

File: advanced/editor-control-identifiers.md (hunk 1 above)
Comment (tylerkelly13) [reply advanced/editor-control-identifiers.md]: yeah, I wasn't sure what would be the least confusing way to order these the table...

Comment (lnewson) [reply advanced/editor-control-identifiers.md]: Yeah I wasn't sure either, but I figured some order was better than no order at all :man_shrugging:

File: advanced/editor-control-identifiers.md [hunk 3]
```diff
@@ -11,187 +11,187 @@ Use the [toolbar]({{ site.baseurl }}/configure/editor-appearance/#toolbar) or th
...
+| print                  | [print]({{ site.baseurl }}/plugins/print/)                             | File                                                                                                     | Prints the current document.                                                                              |
+| searchreplace          | [searchreplace]({{ site.baseurl }}/plugins/searchreplace/)             | Edit                                                                                                     | Opens the search/replace dialog.                                                                          |
+| spellchecker           | [spellchecker]({{ site.baseurl }}/plugins/spellchecker/)               | Tools                                                                                                    | Toggles the spellchecker on/off.                                                                          |
```
Comment (lnewson) [inline advanced/editor-control-identifiers.md]: Not sure if you're worried, but this actually applies to both `spellchecker` and `tinymcespellchecker`.

Comment (tylerkelly13) [reply advanced/editor-control-identifiers.md]: damn.  thanks, thought it might. I'll add a link to the tinymcespellchecker docs too. :)

Comment (tylerkelly13) [reply advanced/editor-control-identifiers.md]: Updated

**Pull request merged**